### PR TITLE
Various minor technical updates

### PR DESCRIPTION
**Pull request by msdemlei**
This PR updates GMS to use some ivoatex luxuries and moves to a somewhat less disagreeable citation style.  It also fixes a few really minor typos.  You may want to throw out the move from verbatim to lstlisting for aesthetic reasons, but I'd be grateful if you didn't.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,4 +25,9 @@ VECTORFIGURES =
 # Additional files to distribute (e.g., CSS, schema files, examples...)
 AUX_FILES =
 
-include ivoatex/Makefile
+-include ivoatex/Makefile
+
+ivoatex/Makefile:
+	@echo "*** ivoatex submodule not found.  Initialising submodules."
+	@echo
+	git submodule update --init
```
Comment (Zarquan) [inline Makefile:33]: It is a good idea to do this, but can we check that this works in `ivoatex-docker` before this is merged into the main branch.
See https://github.com/ivoa/ivoatex-docker/issues/3

Comment (Zarquan) [reply Makefile:33]: Does explicitly adding a new dependency to the build process fall under the heading 'minor technical updates' ?

Comment (Zarquan) [reply Makefile:33]: We should probably check that this works with the automated build process for preview documents.

**Comment by msdemlei** [comment]
On Mon, Sep 13, 2021 at 08:20:38AM -0700, Zarquan wrote:
> @Zarquan commented on this pull request.
> 
> 
> 
> > +ivoatex/Makefile:
> +	@echo "*** ivoatex submodule not found.  Initialising submodules."
> +	@echo
> +	git submodule update --init
> 
> Does explicitly adding a new dependency to the build process fall
> under the heading 'minor technical updates' ?

Well, git, really, is an implicit dependency for something based on
git submodules, so I'd say this isn't adding a new dependency.

By the way, the function of this is to give halfway sane (I'm saying
halfway because of course it's not nice to automatically connect to
adversarial platforms like github, but since we already force people
there I thought keeping this simple is ethically justifiable)
behaviour when people didn't use --recurse-submodules when checking
out a repo.

For other reviewers: zarquan ran into a problem because he's
maintaining a container that processes an externally checked out repo
and hence doesn't have git built in.  However, I'd expect that this
scenario should not be concerned because the git call only happens if
the ivoatex submodule is not present (in which case the build would
already have failed before).

Last point: the github-preview make target already has git calls,
too...

**Comment by msdemlei** [comment]
On Mon, Sep 13, 2021 at 08:18:29AM -0700, Zarquan wrote:
> @Zarquan commented on this pull request.
> 
> 
> 
> > +ivoatex/Makefile:
> +	@echo "*** ivoatex submodule not found.  Initialising submodules."
> +	@echo
> +	git submodule update --init
> 
> It is a good idea to do this, but can we check that this works in `ivoatex-docker` before this is merged into the main branch.
> See https://github.com/ivoa/ivoatex-docker/issues/3

Will you let us know when you're done with that?

File: GMS.tex [hunk 2]
```diff
@@ -45,7 +48,7 @@ \section*{Conformance-related definitions}
...
+Through standard IVOA protocols, many astronomy data centres and institutes offer users access to datasets through Datalink \citep{2015ivoa.spec.0617D} and SODA (Server-side Operations for Data Access, \citet{2017ivoa.spec.0517B}), metadata and catalogs through TAP \citep{2010ivoa.spec.0327D}, and storage through VOSpace \citep{2018ivoa.spec.0621G}.  There are also many instances of custom access services to astronomy resources.  In certain cases the information within these services is proprietary -- it is only allowed to be accessed by certain individuals.  Due to the wide variety and inherently institute-specific set of rules that may define how the information is proprietary, it is beneficial to the owners and maintainers of the rules to have a standard way of describing who has access to what resources.  Additionally, the rules describing resource access may be determined by an entity external to the holder of these resources.  To these ends, this document sets out a standard, programmatic, and interoperable method of determining whether a given user is allowed to access a given proprietary resource.
```
Comment (brianmajor) [inline GMS.tex:52]: Is the `\citet` control element on the SODA citation intentional?

Comment (brianmajor) [reply GMS.tex:52]: I see it's also used below on line 156 so I suppose it is.

**Review by brianmajor — DISMISSED** [review]
Thanks for the review @msdemlei .  I'd like to build from this PR myself, is it complete?

**Comment by brianmajor** [comment]
I was able to build (using the ivoatex container) from this PR without issue.

**Comment by msdemlei** [comment]
On Tue, Sep 14, 2021 at 09:49:00AM -0700, Brian Major wrote:
> @brianmajor approved this pull request.
> 
> Thanks for the review @msdemlei .  I'd like to build from this PR
> myself, is it complete?

If you ask: "Will you add more commits?" I'd say "Not without being
prompted", so I'd say it's about complete.

>> \citep{2015ivoa.spec.0617D} and SODA (Server-side Operations for
>> Data Access, \citet{2017ivoa.spec.0517B}), metadata and catalogs
>> through TAP \citep{2010ivoa.spec.0327D}, and storage through
> 
> Is the `\citet` control element on the SODA citation intentional?

Yes -- \citet is for "citations in running text", which this isn't
quite, but it's close enough.  The *really* correct way to format
such a citation with in-line explanation would be

```
\citep[Server-side Operations for Data Access,][]{2017ivoa.spec.0517B}
```

-- but that's using features I've not yet put into our HTML engine,
so don't do that yet.

Two extra things: you'll want to update the adjacent TAP reference to
2019ivoa.spec.0927D (a.k.a. TAP 1.1).  And you see in this mail why
it's so much better to keep lines shorter than 72 characters even in
TeX source code...

**Review by brianmajor — APPROVED** [review]

**Pull request merged**